Microsoft.Toolkit.Uwp.UI.Controls.DataGrid/Utilities/TypeHelper.cs

```diff
@@ -0,0 +1,432 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should this be in Microsoft.Toolkit Library?

Reply:
Same. We prefered to have a standalone DataGrid assembly.